src/defaults.js

```diff
@@ -2,9 +2,13 @@ import parsers from './parse/parsers';
 
 const defaults = {
   beautifier: {
-    indent_size: 1,
-    indent_char: '	',
-    indent_with_tabs: true
+    indent_size: 2,
+    indent_char: ' ',
+    indent_with_tabs: false,
+    unformatted:
```

Comment:
(one over-engineered solution would be to keep these configs separately in a YAML file and parse 'em. Like I said, over-engineered).